pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -129,6 +132,12 @@ type OFBridge struct {
 	connCh chan struct{}
 	// connected is an internal channel to notify if connected to the OFSwitch or not. It is used only in Connect method.
 	connected chan bool
+	// pktConsumers is a map from PacketIn reason to the channel that is used to publish the PacketIn message.
+	pktConsumers map[uint8]chan *ofctrl.PacketIn
+	// tlvMapStatus is the status of tlv-map table. It is used to allocate new tlv-map which is used in the Geneve header.
+	tlvMapStatus *ofctrl.TLVTableStatus
```

Comment:
I feel like I am missing something because I do not understand why `tlvMapStatus` is part of Antrea. Shouldn't ofnet take care of this behind the scenes? It seem that we use it it `AddTLVMap` to ensure that the OVS tlv-map is configured consistently (e.g. we do not try to re-use the same `tuneMetadataX` field for another TLV Class. But I do not understand why it is the responsibility of Antrea to do this.